test/LaTeXAndHTML/succeed/Issue7815.agda

```diff
@@ -0,0 +1,6 @@
+module Issue7815 where
+module X where
+module _
+  (a : Set)
+  (let module Y = X)
+  where
```

Comment:
Would you mind to add some comments to this test case what is tested here and what is important.
E.g. "before fix of #7815, highlighting was missing from `a : Set`" or something like this.
This insures (somewhat) against accidental breakage of the test.